### PR TITLE
Allows pAI's to use *beep.

### PR DESCRIPTION
**Pull request by Spacemanspark**
-Allows pAI's to use *beep as well.

@Aurorablade @TheDZD

### CODE REVIEW

File: code/modules/mob/living/silicon/pai/emote.dm [hunk 1]
```diff
@@ -57,6 +57,23 @@
...
+					if (param == A.name)
+						M = A
+						break
+			if(!M)
+				param = null
+
+			if (param)
+				message = "<B>[src]</B> pings at [param]."
+			else
+				message = "<B>[src]</B> pings."
```
Comment (veganzombeh) [inline code/modules/mob/living/silicon/pai/emote.dm]: Shouldn't this be `beeps`?

Comment (Spacemanspark) [inline code/modules/mob/living/silicon/pai/emote.dm]: WHOOPS.

**Comment by veganzombeh** [comment]
You should probably add beep to the list of emotes for *help too.

**Comment by Spacemanspark** [comment]
Thanks, see any other issues?

**Comment by tigercat2000** [comment]
Should really just move *beep *ping and *buzz to silicon/emote

**Comment by TheDZD** [comment]
Yeah, pretty much that @tigercat2000 said.

**Comment by Ultimate-Chimera** [comment]
Let AI's do it too.

**Comment by Fox-McCloud** [comment]
yeah, move this to silicons and ensure that they all obey the cooldown.

**Comment by Markolie** [comment]
DNM until it's moved to silicon emotes.

**Comment by Fox-McCloud** [comment]
How goes this, @Spacemanspark ?

**Comment by Spacemanspark** [comment]
Posting from my phone, computer is BSODing with the error DPC WATCHDOG VIOLATION.
Can't work on this right now.

**Comment by Spacemanspark** [comment]
Right, I think I'll leave this to someone else.

**Pull request closed without merging**